libs/client-shared/src/lib/components/text-area/text-area.component.ts

```diff
@@ -20,18 +20,24 @@ import { FormItemWrapperComponent } from '../form-item-wrapper';
     },
   ],
 })
-export class TextAreaComponent implements ControlValueAccessor {
+export class TextAreaComponent implements ControlValueAccessor, AfterViewInit {
   @Input() public title = '';
   @Input() public value = '';
-  @Input() public icon = '';
+  @Input({ transform: coerceBooleanProperty }) public isRequired = '';
```

Comment:
This should be a boolean property.
```suggestion
  @Input({ transform: coerceBooleanProperty }) public isRequired = false;
```